### DISCUSSION TITLE
Customisable packege

### DISCUSSION THREAD

**Discussion by memeLordo** [mason-org/mason.nvim#1337] [Q&A]
Is there any way to customise my package settings for Mason plugin?
Like, for example, setting my linters to ignore some type of errors/warnings.
If this feature exists, please note me how.

**Comment by williamboman** [comment]
Mason only installs programs on your system. Each package may have their own capabilities for customizing behaviour. Commonly this is done through configuration files, such as `pyproject.tomll` or `.pretterrc`, etc.
